### PR TITLE
SAK-28415 - Update SignUp datepicker

### PR DESCRIPTION
**Pull request by mpellicer**
Update SignUp with the new date picker introduced in SAK-23662

### CODE REVIEW

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java [hunk 1]
```diff
@@ -641,6 +673,27 @@ public void validateNewMeeting(ActionEvent e) {
...
+			String isoEndTime = params.get(HIDDEN_ISO_ENDTIME);
+
+			if(Utilities.isValidISODate(isoEndTime)){
+				this.signupMeeting.setEndTime(Utilities.parseISODate(isoEndTime));
+			}
+
+			String isoUntilTime = params.get(HIDDEN_ISO_UNTILTIME);
+
+			if(Utilities.isValidISODate(isoUntilTime)){
+				System.out.println("Configuro "+Utilities.parseISODate(isoUntilTime));
```
Comment (ern) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java]: Don't use System.out.println

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/CopyMeetingSignupMBean.java [hunk 2]
```diff
@@ -331,6 +340,26 @@ public String processSaveCopy() {
...
+		String isoEndTime = params.get(HIDDEN_ISO_ENDTIME);
+
+		if(Utilities.isValidISODate(isoEndTime)){
+			this.signupMeeting.setEndTime(Utilities.parseISODate(isoEndTime));
+		}
+
+		String isoUntilTime = params.get(HIDDEN_ISO_UNTILTIME);
+
+		if(Utilities.isValidISODate(isoUntilTime)){
+			System.out.println("Configuro "+Utilities.parseISODate(isoUntilTime));
```
Comment (ern) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/CopyMeetingSignupMBean.java]: System.out.println

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java (hunk 1 above)
Comment (mpellicer) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java]: Wasn't on purpose, just a mistake.

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/CopyMeetingSignupMBean.java (hunk 2 above)
Comment (mpellicer) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/CopyMeetingSignupMBean.java]: Wasn't on purpose, just a mistake.

**Pull request merged**